tests/serverpod_test_server/test_integration/auth_module/auth_signout_test.dart

```diff
@@ -0,0 +1,219 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_auth_server/serverpod_auth_server.dart';
+import 'package:test/test.dart';
+import '../test_tools/serverpod_test_tools.dart';
+
+void main() {
+  withServerpod(
+    'Given an authenticated user',
+    (sessionBuilder, _) {
+      late String userName;
+      late String email;
+      late String password;
+      late Session session;
+
+      setUp(() async {
+        session = sessionBuilder.build();
+        userName = 'test';
+        email = 'test@serverpod.dev';
+        password = 'password';
+        await Emails.createUser(session, userName, email, password);
+      });
+
+      test(
```

Comment:
Suggestion: Please break this up into smaller validations and have the `when` part on a group level.

Reply:
Same for the other tests.